fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadParam.java

```diff
@@ -47,6 +48,8 @@ public class StreamLoadParam {
     private static final String IDLE_TRANSACTION_TIMEOUT = "idle_transaction_timeout";
     private static final String PARTIAL_UPDATE_MODE = "partial_update_mode";
 
+    private static final String ENVELOPE = "envelope";
```

Comment:
> Can we just define a single ENVELOP const String, and use it anywhere?

We define ENVELOP const String in this java class right now, it will be used anywhere.